custom_components/mqtt_vacuum_camera/common.py

```
@@ -133,3 +135,36 @@ def is_rand256_vacuum(vacuum_device: DeviceEntry) -> bool:
+                try:
+                    if "." in my_payload:
+                        return float(my_payload)
+                    else:
+                        return int(my_payload)
+                except ValueError:
+                    pass
+            return my_payload
+        elif isinstance(my_payload, (int, float)):
+            return my_payload
+        elif isinstance(my_payload, bytes):
+            return my_payload
```

Ruff

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)